faqs/Tech/what-is-musd.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: what-is-musd
+sidebar_position: 2
```

Comment:
-* This needs to match the ordering on the FAQ page.

You put the MUSD article first in this section, which is fine, but it means you also need to bump down the "sidebar position" of all the other articles so they stay equivalent.

Reply:
Fixed.